tooling/util/package.js

```diff
@@ -17,7 +17,9 @@ const cwd = 'packages';
 exports.list = async function list() {
   const packages = await g('**/package.json', {cwd});
 
-  return packages.map((p) => path.dirname(p)).filter((p) => (p.startsWith('webex') || p.startsWith('@webex') || p.startsWith('calling')));
+  return packages
+    .map((p) => path.dirname(p))
+    .filter((p) => p.startsWith('webex') || p.startsWith('@webex') || p.startsWith('calling'));
```

Comment:
Same here. Can we revert the formatting changes like this from the changeset? Just to avoid confusions